cmd/lambda/leases/create.go

```diff
@@ -29,10 +28,11 @@ type createLeaseRequest struct {
 func CreateLease(w http.ResponseWriter, r *http.Request) {
 
 	c := leaseValidationContext{
-		maxLeaseBudgetAmount:     aws.Float64Value(maxLeaseBudgetAmount),
```

Comment:
no need to use the aws pkg to dereference pointer, can just use built-in golang syntax.